### ISSUE TITLE
pylint 2.5.2 throws 'unable to import ...'

### ISSUE THREAD

**Issue by gc-plp** [firecracker-microvm/firecracker#1913]
When building a new Docker container to use with the `devtool` script, `pip3` pulls in a newer `pylint` version that throws import errors:

```
************* Module integration_tests.functional.test_initrd
integration_tests/functional/test_initrd.py:5:0: E0401: Unable to import 'framework.microvm' (import-error)
```
`pylint` version that throws the error above is `2.5.2`.
`pylint` version that is included in the `v14` container and does not throw the error above is `2.4.4`

To fix this we should address the import errors and evaluate the possibility of using fixed package versions.

**Comment by gc-plp** [comment]
To test the issue described, one can use the `v15` container version, by applying the following patch:
```diff
diff --git a/tools/devtool b/tools/devtool
index 7a53a267..6297337d 100755
--- a/tools/devtool
+++ b/tools/devtool
@@ -73,7 +73,7 @@
 # Development container image (name:tag)
 # This should be updated whenever we upgrade the development container.
 # (Yet another step on our way to reproducible builds.)
-DEVCTR_IMAGE="fcuvm/dev:v14"
+DEVCTR_IMAGE="fcuvm/dev:v15"
 
 # Naming things is hard
 MY_NAME="Firecracker $(basename "$0")"
```

**Comment by andreeaflorescu** [comment]
How about we remove some of the python linters instead? Don't we have multiple? Are they all needed?

**Comment by gc-plp** [comment]
Currently we use `pylint`, `pycodestyle` and `pydocstyle`.
`pylint` seems to be one of the more mature ones, so I would be inclined to either keep it, or replace it with something else, such as `flake8`.

**Comment by andreeaflorescu** [comment]
Can we remove the other ones then? If I remember correctly we recently had to fix something about one of the others.

**Comment by gc-plp** [comment]
Yes, but we should do this in the context of a separate issue - see #1914

**Comment by lauralt** [comment]
Fixed by #1917
